src/ints/bios_disk.cpp

```diff
@@ -326,7 +332,11 @@ static bool has_image(const std::array<T, N> &arr) {
 	return std::any_of(std::begin(arr), std::end(arr), to_bool);
 }
 
-static Bitu INT13_DiskHandler(void) {
+void diskio_delay(Bits value /*bytes*/, DiskNoiseDevice* disknoise,
+                  DiskType type = DiskType::Floppy);
```

Comment:
Don't call functions directly like this across module boundaries, please. You're in the "interrupts" module now, and you're calling something from the "DOS" module. So the proper way is to expose this in the DOS modules's API in `include/dos_inc.h` or `include/dos_system.h` like this:

```cpp
void DOS_PerformeDiskIoDelay(...)
```

Then call that from here.